src/theory/strings/sequences_rewriter.h

```diff
@@ -309,6 +311,8 @@ class SequencesRewriter : public TheoryRewriter
   ArithEntail d_arithEntail;
   /** Instance of the entailment checker for strings. */
   StringsEntail d_stringsEntail;
+  /** Whether to use regular expression inclusion tests. */
+  bool d_useRegexpInclusion;
```

Comment:
I'd prefer if this wasn't an option on the rewriter, and instead guards the portion of the RegExpSolver that does the inclusion tests, isn't this easier to do?

Reply:
Discussed offline and addressed.